jck/runtime.api/playlist.xml

```diff
@@ -1219,10 +1219,7 @@
 		<disables>
 			<disable>
 				<comment>backlog/issues/567</comment>
-				<impl>openj9</impl>
-			</disable>
-			<disable>
-				<comment>backlog/issues/567</comment>
+				<version>8</version>
 				<impl>ibm</impl>
 			</disable>
 		</disables>
```

Comment:
```suggestion
		<disables>
			<disable>
				<comment>backlog/issues/567</comment>
				<version>8</version>
				<impl>openj9</impl>
			</disable>
		</disables>
```

Reply:
Only zos uses `ibm` atm

Reply:
JDK8 OpenJ9 works fine, only ibm implementation of JDK8 fails. If only JDK11 zos uses ibm impl, then we can enable this test for all releases. Thank you.

Failing combination is only, rest of them can be enabled.
JDK8 - ibm impl for all platforms.

Reply:
The playlist is for Jenkins test jobs. In other words, since we do not have JDK8 ibm builds on Jenkins, the exclusion of `JDK8 ibm` does not make sense. 
If you know of any JDK8 ibm builds on Jenkins, please let us know. Thanks.

Reply:
I've modified playlist which removed JDK8 ibm and added JDK11 ibm(zos). Please check the latest commit. Thanks.

Reply:
Thanks for the update @psoujany . If we just want to exclude for zos, could you put `<platform>.*zos.*</platform>` instead? This will be more clear as this is zos specific exclusion. Thanks.

```
		<disables>
			<disable>
				<comment>backlog/issues/567</comment>
				<platform>.*zos.*</platform>
			</disable>
		</disables>
```

Reply:
Thanks @llxia , as per your suggestion I modified playlist. Could you please review latest commit. Thanks.